calcom/cal.com
# fix(app-store): Giphy Ux fixes

## What does this PR do?
Fixes UX/UI as per Figma designs

Fixes #2682 



## Type of change




## How should this be tested?

    Install the Giphy app from App Store
    Create a new event or use any existing event type
    Click on "Show advanced settings" & scroll to the bottom
    Press search on the Giphy button which should open a popup
    On the popup type something in the search box and press search
    You can use the next & prev icons to see more gifs
    After selecting one press Save
    the selected Gif should be visible on the event edit page
    Press the update button on the page
    Preview this event and schedule a time, the gif should be visible on the success page
   
   To Search By URL use the gif URL from here 
<img width="287" alt="image" src="https://user-images.githubusercontent.com/25253907/167938597-86eefe80-957d-4607-822e-f1b43e61f1e4.png">

# Screenshots

<img width="840" alt="Screenshot 2022-05-17 at 4 54 00 PM" src="https://user-images.githubusercontent.com/25253907/168800114-da1ad572-584a-4f1b-a9ef-ce189d2544b7.png">
<img width="765" alt="Screenshot 2022-05-17 at 4 53 47 PM" src="https://user-images.githubusercontent.com/25253907/168800126-f4c66593-9332-4c8d-ad38-80f5c670fc94.png">

<img width="895" alt="Screenshot 2022-05-17 at 4 53 41 PM" src="https://user-images.githubusercontent.com/25253907/168800132-e9a44f35-2d44-48db-a584-3c61bb5baedb.png">
